10-flow-rate-monitor/firmware/README.md

```diff
@@ -0,0 +1,42 @@
+# Flow Rate Sensor
```

Comment:
This isn’t essential, but it might be worth copying over NF9’s firmware README, which more detailed descriptions of these steps.

Reply:
Agreed - integrating NF9 instructions into NF10 readme